scenes/lessons/7th_class/a_b+c+d/a_b+c+d.py

```diff
@@ -0,0 +1,616 @@
+from manim import WHITE, YELLOW, ORANGE, BLUE, RED, MAROON, GOLD_A, TEAL, GREEN
+from manim import MathTex, VGroup
+from manim import SurroundingRectangle, Rectangle, Circle
+from manim import RIGHT, LEFT, UP, DOWN, DR, UL, DR, ORIGIN
+from manim import AnimationGroup, Write, Unwrite, FadeIn, FadeOut, Create, ReplacementTransform
+from manim import Indicate, Circumscribe, Wiggle, TransformMatchingTex, ClockwiseTransform
+
+import numpy as np
+from colour import Color
+
+from aramanim import ConectionLine
+from hanrahashiv import FormulaModificationsScene, ModifyFormula
+
+BIG_SCALE = 1.3
+SCALE = .45
+
+
+def get_tex(expression, color=WHITE):
```

Comment:
I can help you generalize this, if you want.

Reply:
sure

Reply:
@tmargary , երբ պետքական թվա ընդհանրացնելը, սա, իրար հետ նայենք կամ ես գրեմ